extensions/data-workspace/src/dialogs/newProjectDialog.ts

```diff
@@ -284,7 +284,6 @@ export class NewProjectDialog extends DialogBase {
 		this.formBuilder = view.modelBuilder.formContainer().withFormItems([
 			{
 				title: constants.TypeTitle,
-				required: true,
```

Comment:
I removed the required from this component because radio buttons and radio card groups don't have the required property, I'm guessing because one of the options is always selected so it isn't something the user has to actively fill out.